### PR TITLE
add napoleon ext

### LINKED ISSUE #102: Google docstring standard not rendering correctly.
@ocefpaf I tested out the docstring on my latest change #98. But `Args` doesn't seem to be rendering correctly.. Should I add `*` or `-` to make it a list?

screenshot 2017-09-29 08 26 48

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #102

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'sphinx.ext.napoleon',
```
Comment (ocefpaf) [inline docs/source/conf.py]: @lsetiawan I forgot that `napoleon`, the extension for the numpy/scipy and google style docstrings, is builtin and we still need to enable it.

Comment (lsetiawan) [reply docs/source/conf.py]: Thanks for catching that! :)

File: docs/source/modules.rst [hunk 2]
```diff
@@ -5,3 +5,11 @@
    :members:
    :undoc-members:
    :show-inheritance:
+
+:mod:`Read Services`
+----------------------
+
+.. automodule:: odm2api.ODM2.services.readService
```
Comment (lsetiawan) [inline docs/source/modules.rst]: I haven't fixed all the docstring for this module yet. It's gonna be a mess except for one function `getAffiliations`. But I guess for now that's fine. The current docstrings still rendered, at lease now I can see it without going to the actual module python file.

Comment (ocefpaf) [reply docs/source/modules.rst]: You don't need to merge this. I only sent the PR to show you with code that what was wrong was not your docstring :smile:

Comment (lsetiawan) [reply docs/source/modules.rst]: Could you maybe remove this change and keep the other one to `conf.py` so I can merge? Thanks 😄

**Comment by lsetiawan** [comment]
It works now with the built in napoleon extension! 🎉

File: docs/source/modules.rst (hunk 2 above)
Comment (ocefpaf) [reply docs/source/modules.rst]: Done!

**Pull request merged**